mlblocks_primitives/mlprimitives.timeseries.aggregate_average_time.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "mlprimitives.timeseries.aggregate_average_time",
+    "author": "Ihssan Tinawi <itinawi@mit.edu>",
+    "description": "mlprimitives.timeseries.aggregate_average_time",
+    "classifiers": {
+        "type": "preprocessor",
+        "subtype": "feature_extractor"
+    },
+    "modalities": ["timeseries"],
+    "primitive": "mlprimitives.timeseries.aggregate_average_time",
+    "produce": {
+        "args": [
+            {
+                "name": "df_time_value",
+                "type": "Pandas.DataFrame"
+            },
+            {
+                "name": "interval_time_delta",
+                "type": "int"
+            },
+            {
+                "name": "start_time",
```

Comment:
This and end_time should be set as fixed hyperparameters with `null` as default.